### PR TITLE
DOC - Update homepage to include project details

### PR DESCRIPTION
**Pull request by pavithraes**
Addresses #397

## Description

This pull request:

- Explains the three conda-store projects on the homepage

https://github.com/conda-incubator/conda-store/assets/33131404/150d1bf2-6b5c-4ef2-a050-4084f0c40ce9

## Pull request checklist

- [x] Did you test this change locally?
- [x] Did you update the documentation (if required)?
- [ ] Did you add/update relevant tests for this change (if required)?

### CODE REVIEW

File: docusaurus-docs/src/pages/index.js [hunk 1]
```diff
@@ -31,66 +31,129 @@ function HomepageHeader() {
...
-        Admins have role-based access management, to allow users to share environments across (and only with) their team.
+        As admins, manage users and teams, and approve packages and channels, to maintain organizational standards.
+      </>
+    ),
+  },
+  {
+    title: '💻 System Agnostic',
+    description: (
+      <>
+        Run conda-store on any major cloud provider, on-prem, or on local machines.
```
Comment (pavithraes) [inline docusaurus-docs/src/pages/index.js:109]: Unsure

File: docusaurus-docs/src/pages/index.js [hunk 2]
```diff
@@ -31,66 +31,129 @@ function HomepageHeader() {
...
+        User-friendly frontend to access conda-store features in a React application
+      </>
+    ),
+  },
+  {
+    title: '🪐 JupyterLab extension',
+    link: '/jupyterlab-conda-store/introduction',
+    description: (
+      <>
+        Provides conda-store-ui frontent within the JupyterLab interface
```
Comment (kcpevey) [inline docusaurus-docs/src/pages/index.js:58]:
```suggestion
    JupyterLab interface that provides conda-store-ui frontend
```
Changed it to have the same structure as other two

File: docusaurus-docs/src/pages/index.js [hunk 3]
```diff
@@ -31,66 +31,129 @@ function HomepageHeader() {
...
+      <>
+        Create, update, and manage environments using a user-friendly graphical UI or YAML editor, available from within JupyterLab or standalone.
+      </>
+    ),
+  },
+  {
+    title: '📋 Reproducible Artifacts',
+    description: (
+      <>
+        Share fully-reproducible environments with auto-generated artifacts like lockfiles, YAML files, docker images, or tarballs.
```
Comment (pavithraes) [inline docusaurus-docs/src/pages/index.js:77]:
```suggestion
        Share fully-reproducible environments with auto-generated artifacts like lockfiles, YAML files, docker images, and tarballs.
```

File: docusaurus-docs/src/pages/index.js (hunk 1 above)
Comment (pavithraes) [reply docusaurus-docs/src/pages/index.js:109]: We have a kubernetes-based installation, so I'm happy to say this

**Review by kcpevey — APPROVED** [review]

**Pull request merged**